mac

```diff
@@ -238,6 +239,9 @@ if [ ! $SKIP_HOMEBREW_UPDATE ]; then
     # Terraform
     brew "hashicorp/tap/terraform"
 
+    # Websockets
+    brew "anycable-go"
```

Comment:
Since we no longer run legacy versions of `anycable-go`, we should install it via homebrew